extensions/amp-playbuzz/0.1/validator-amp-playbuzz.protoascii

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-playbuzz
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "SCRIPT"
+  spec_name: "amp-playbuzz extension .js script"
```

Comment:
For new tags, we want to add a requirement that if the .js file is present, a corresponding tag is also present, to avoid javascript being loaded which isn't used, a performance anti-pattern. This is fairly new, so most tags don't have it, but an example would be https://github.com/ampproject/amphtml/blob/master/extensions/amp-reddit/0.1/validator-amp-reddit.protoascii

Can you add here:

also_requires_tag: "amp-playbuzz"